Composer/packages/client/src/recoilModel/dispatchers/__tests__/qna.test.tsx

```diff
@@ -95,7 +95,7 @@ describe('QnA dispatcher', () => {
       });
     });
 
-    expect(renderedComponent.current.qnaFiles[0].content).toContain(content);
+    expect(renderedComponent.current.qnaFiles[0].content.replace(/\s/g, '')).toContain(content.replace(/\s/g, ''));
```

Comment:
is there a reason why this test changed?

Reply:
I don't know why it happened, but the results were coming out with a space before each newline, so I decided to just squish all the whitespace out.